src/prommis/nanofiltration/diafiltration_solute_properties.py

```diff
@@ -58,32 +57,40 @@ def build(self):
             },
         )
 
-        # add molecular weight
-        self.molar_mass = Param(
+        # add thermal reflection coefficient, related to solute rejection
+        self.sigma = Param(
             self.component_list,
-            units=units.kg / units.mol,
+            units=units.dimensionless,
             initialize={
-                "Li": 0.006941,
-                "Co": 0.05893,
-                "Cl": 0.03545,
+                "Li": 1,
+                "Co": 1,
+                "Cl": 1,
             },
         )
 
-        # add thermal reflection coefficient, where 1 represents ideal behavior
-        self.sigma = Param(
+        # add partition coefficient
+        # currently H,Li is based on https://doi.org/10.1021/acs.iecr.4c04763
+        # H,Co and H,Cl are arbitrarily chosen to be the same value
+        self.partition_coefficient = Param(
             self.component_list,
             units=units.dimensionless,
             initialize={
-                "Li": 1,
-                "Co": 1,
-                "Cl": 1,
+                "Li": 0.3,
+                "Co": 0.3,
+                "Cl": 0.3,
             },
+            domain=NonNegativeReals,
```

Comment:
I *strongly* discourage using Pyomo domains for anything other than specifying integer variables. Otherwise, Pyomo silently truncates the bounds to be within the domain if the user tries to change them. Even if we expect a variable to always be nonnegative, we might want to relax the lower bound in order to prevent IPOPT from treating the lower bound as an active constraint.